core/src/main/java/net/akami/yggdrasil/item/ElementItem.java

```diff
@@ -59,6 +62,7 @@ public void onRightClicked(CancellableEvent<?> event, GameItemClock clock) {
 
     private void click() {
         user.currentSequence().add(getType());
+        textDisplayer.display(getName());
```

Comment:
As @Tran-Antoine said, you can use some special caracters to make icons for the sequence instead of element's name:
- Fire: 🔥
- Air: ☴
- Earth: ⏚
- Water: 🌊
(Icons can be changed using resources pack)